test/e2e/storage/persistent_volumes-local.go

```diff
@@ -490,19 +493,27 @@ func makeLocalPVConfig(config *localTestConfig, volume *localTestVolume) framewo
 func createLocalPVCPV(config *localTestConfig, volume *localTestVolume) {
 	pvcConfig := makeLocalPVCConfig(config)
 	pvConfig := makeLocalPVConfig(config, volume)
-
 	var err error
 	volume.pv, volume.pvc, err = framework.CreatePVPVC(config.client, pvConfig, pvcConfig, config.ns, true)
 	framework.ExpectNoError(err)
 	framework.ExpectNoError(framework.WaitOnPVandPVC(config.client, config.ns, volume.pv, volume.pvc))
 }
 
 func makeLocalPod(config *localTestConfig, volume *localTestVolume, cmd string) *v1.Pod {
-	return framework.MakePod(config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, cmd)
+	return framework.MakeSecPod(config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, cmd, false, false, selinuxLabel)
+}
+
+func createSecPod(config *localTestConfig, volume *localTestVolume, hostIPC bool, hostPID bool, seLinuxLabel *v1.SELinuxOptions) (*v1.Pod, error) {
```

Comment:
Who calls this?

Reply:
Bu default PODs are created with the same SELinux labels, this call is used for that createLocalPod, as you can see there is no SELinux label passed as a parameter. The reason is I did not want to touch every other functions using it.  So under the hood, default SELinux is passed. For the case when POD needs an explicit SELinux label different from default, then new function createSecPod should be used and one of its parameters is SELinux.

Reply:
I only see calls to `CreateSecPod`, but not `createSecPod`

Reply:
Right, currently it is not called, it was called by negative test which is gone now. I wanted to keep it for future so whoever need it does not need to reinvent anything.

Reply:
Ok, can you add a comment explaining the difference

Reply:
Sure thing